### PR TITLE
Update GRCh38_Myeloma_FISH_Probe_Locations.bed

### PR DESCRIPTION
**Pull request by bryce-turner**
Adding tentative location updates. Do not merge until we fix the gain/loss for new regions.

### CODE REVIEW

File: myeloma_FISH_probe_locations/GRCh38_Myeloma_FISH_Probe_Locations.bed [hunk 1]
```diff
@@ -25,7 +26,8 @@ chr10	94240243	95240243	10q23	gain
 chr11	17978453	18978453	11p15	gain
 chr11	102317450	102378670	BIRCs	loss
 chr11	110129275	112129277	11q23	gain
-chr12	11847066	12847066	12p13	loss
+chr12	11847066	12816446	12p13	loss
```
Comment (bryce-turner) [inline myeloma_FISH_probe_locations/GRCh38_Myeloma_FISH_Probe_Locations.bed:29]: Adjusted region to avoid overlap with GPRC5D

Comment (PedalheadPHX) [reply myeloma_FISH_probe_locations/GRCh38_Myeloma_FISH_Probe_Locations.bed:29]: I would not alter existing regions
BCMA, GPRC5D measure should be loss not gain
FCRL5 measure, should also be loss

Comment (bryce-turner) [reply myeloma_FISH_probe_locations/GRCh38_Myeloma_FISH_Probe_Locations.bed:29]: How would we like to handle the overlapping regions? Allow the overlap or adjust the region for the new addition?

**Review by PedalheadPHX — APPROVED** [review]
Looks good will merge

**Pull request merged**